low-level/framework/utils/salt_util.py

```diff
@@ -132,7 +132,8 @@ def get_consul_vip(self, _err=None):
             try:
                 if self.pillar_info is not None:
                     data_store_config = \
-                        self.pillar_info[self.node_id][self.SSPL_KEY][self.DATASTORE_KEY]
+                        self.pillar_info[self.node_id][self.CLUSTER_KEY][self.node_id]\
+                        ['network']['data_nw']['roaming_ip']
                     if data_store_config is not None:
```

Comment:
You might need a change here, since I think above command will return you the vip.